src/server/config/schema.js

```diff
@@ -29,6 +29,7 @@ module.exports = Joi.object({
     port: Joi.number().default(5601),
     autoListen: Joi.boolean().default(true),
     defaultRoute: Joi.string(),
+    basePath: Joi.string().default('').allow('').regex(/[^\/]$/, 'no trailing slashes'),
```

Comment:
Should this make sure there's a leading slash too?  Paths are off if it's omitted, it ends up looking for something like localhost/basepath/app/basepath